### ISSUE TITLE
Option to disregard Lap 0 time in Total Time Computation

### ISSUE THREAD

**Issue by DevnathNair** [RotorHazard/RotorHazard#133]
As of the latest version, total times are computed by summing up times, starting from the start tone (ie. Lap(0)+Lap(1)+Lap(2)+...Lap(n)). 

_Lap 0 being the time between the start tone and the first gate pass_

In our local race format, we time qualifiers by measuring total time starting after a first gate pass. This is such so that staggered launches are facilitated to avoid mid-airs.

An option to disregard the Lap 0 time (perhaps in the race format tab) in computing the total time would allow for extracting total times without having to manually add subsequent recorded lap times. (ie. Lap(1)+Lap(2)+...Lap(n))

**Comment by HazardCreative** [comment]
Yes, I was thinking about this a while back and it makes sense to add. Once consideration is how races run this way should be populated in the event leaderboard. Our group doesn't use the total time stat from the event leaderboard so I don't have a strong opinion.

**Comment by DevnathNair** [comment]
The Leaderboard instead of being populated by the maximum number of lap times as it is currently should be switched over to one that ranks pilots by the fastest total time in the format that we follow.

Our race format is quite straightforward: 

1) _Fixed Number of Laps per Heat_
2) _Fixed Number of Heats_
3) _Total Time is Computed for each Heat_
4) _Pilots are ranked by the fastest Total Time out of all Heats_
5) _Further Seeded into a New Class and the format repeated until the top 3 fastest pilots are arrived at_

tldr; The only change required to the leaderboard would be that pilots need to be ranked based on the _fastest total time_ instead of number of laps in this specific format. The fastest laptimes can still remain and are relevant in this format.

**Comment by HazardCreative** [comment]
Could be something like a class win condition and event win condition are needed, but that would be a topic for a separate issue.

**Comment by DevnathNair** [comment]
I believe so, yes.
